tests/Tests.AzureAppConfiguration/JsonContentTypeTests.cs

```diff
@@ -28,20 +28,15 @@ public void JsonContentTypeTests_CompareJsonSettingsBetweenAppConfigAndJsonFile(
 
             var appconfigSettings = new ConfigurationBuilder()
                 .AddAzureAppConfiguration(options => options.ClientManager = mockClientManager)
-                .Build()
-                .AsEnumerable();
+                .Build();
 
             var jsonSettings = new ConfigurationBuilder()
                 .AddJsonFile(jsonFilePath)
-                .Build()
-                .AsEnumerable();
-
-            Assert.Equal(jsonSettings.Count(), appconfigSettings.Count());
+                .Build();
 
-            foreach (KeyValuePair<string, string> jsonSetting in jsonSettings)
+            foreach (KeyValuePair<string, string> jsonSetting in jsonSettings.AsEnumerable())
             {
-                KeyValuePair<string, string> appconfigSetting = appconfigSettings.SingleOrDefault(x => x.Key == jsonSetting.Key);
-                Assert.Equal(jsonSetting, appconfigSetting);
+                Assert.Equal(jsonSettings.GetSection(jsonSetting.Key).Value, appconfigSettings.GetSection(jsonSetting.Key).Value);
```

Comment:
`GetSection` can return null.

Reply:
I thought it couldn't based on [this](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.configuration.iconfiguration.getsection?view=dotnet-plat-ext-8.0#remarks).

Reply:
You're right, I accidentally opened the wrong "[GetSection](https://learn.microsoft.com/en-us/dotnet/api/system.configuration.configuration.getsection?view=dotnet-plat-ext-8.0#returns)" :)